extensions/debezium-postgres/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -24,8 +24,12 @@
 name: "Camel Debezium PostgresSQL Connector"
 description: "Capture changes from a PostgresSQL database"
 metadata:
-  unlisted: true
-  guide: "https://camel.apache.org/components/latest/debezium-postgres-component.html"
+  keywords:
+  - "camel"
+  - "database"
+  - "postgres"
+  - "sql"
+  guide: "https://quarkus.io/guides/camel"
   categories:
   - "integration"
-  status: "preview"
+  status: "stable"
```

Comment:
Could you please re-generate this file by running `mvn -N cq:update-quarkus-metadata` from the repo root folder?